cluster/photon-controller/setup-prereq.sh

```diff
@@ -118,40 +134,80 @@ function create-vm-flavor {
     local flavor_name=$1
     local flavor_spec=$2
     local rc=0
+    local output
 
     $PHOTON flavor list | grep "\t${flavor_name}\t" > /dev/null 2>&1 || rc=$?
```

Comment:
May be you want to warn about error as well. These are flavor states in photon:
CREATING,
  READY,
  PENDING_DELETE,
  ERROR,
  DELETED;